app/requirements.txt

```diff
@@ -48,7 +48,7 @@ scikit-build
 sentry-sdk>=0.10.2
 sentry-sdk[flask]>=0.10.2
 
-setuptools>=42
+setuptools>=45
```

Comment:
Did an error prompt this or just keeping current?

Reply:
An error, I ended up pulling in a versions of setuptools_scm (below) that required setuptools 45 and the above line was getting 44 and it all failed.